showtime-xyz/showtime-frontend
# fix: video and placehoder image overlapping

# Why

- video and image overlapping.
- some components have type error/warning
- we removed the Collection component, so we need to adjust item height.
- the style of the card is inconsistent with Figma.
- mint modal style has some broken.
# How

- blur the background image and center the video, and add a patch for `expo-av`.
- fix many types of errors.
- adjust card item height.
- improve card style.
- fix mint modal style.
# Test Plan

- check display normally if the video and image overlapping.
- check if the card style is looking good.
- check if the mint modal is looking good.
